src/data-fetchers/bigwig/bigwig-data-fetcher.ts

```diff
@@ -86,7 +86,7 @@ function BigWigDataFetcher(HGC: import('@higlass/types').HGC, dataConfig: BigWig
             this.dataPromises.push(this.loadBBI(dataConfig));
         }
 
-        loadBBI(dataConfig: { url: string }) {
+        async loadBBI(dataConfig: { url: string }) {
```

Comment:
I assume this change was needed by some recent changes in BBI?

Reply:
Not exactly. This function returns either a promise or `null`, and we push the results to `dataPromises`. Previously the return type was `Promise<void> | null`. The async makes it always return a Promise.

```ts
        async loadBBI(dataConfig: { url: string }) {
            if (dataConfig.url) {
                this.bwFile = new BigWig({
                    filehandle: new RemoteFile(dataConfig.url)
                });
                return this.bwFile.getHeader().then((h: BigWigHeader) => {
                    this.bwFileHeader = h;
                });
            } else {
                console.error('Please enter a "url" field to the data config');
                return null;
            }
        }
```

Reply:
I see – Good to know that.